android/app/src/main/java/app/organicmaps/sdk/OrganicMaps.java

```diff
@@ -160,7 +159,8 @@ private void initNativePlatform() throws IOException
     createPlatformDirectories(writablePath, privatePath, tempPath);
 
     nativeInitPlatform(mContext, apkPath, writablePath, privatePath, tempPath, app.organicmaps.BuildConfig.FLAVOR,
-                       app.organicmaps.BuildConfig.BUILD_TYPE, UiUtils.isTablet(mContext));
+                       app.organicmaps.BuildConfig.BUILD_TYPE,
+                       /* isTablet */ false);
```

Comment:
- isTablet not used?
-  Maybe ```false /* isTablet */```

Reply:
https://github.com/organicmaps/organicmaps/pull/10979#issuecomment-3134886235